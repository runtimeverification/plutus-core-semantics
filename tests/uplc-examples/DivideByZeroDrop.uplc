(program
  1.0.0
  [
    [ (force (force (delay (delay (lam x (lam y x)))))) (con integer 0) ]
    [ [ (builtin divideInteger) (con integer 0) ] (con integer 0) ]
  ]
)
