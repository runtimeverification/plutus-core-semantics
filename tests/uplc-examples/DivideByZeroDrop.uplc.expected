An error has occurred:  User error:
The provided Plutus code called 'error'.
Caused by: (divideInteger 0 0)
